vignettes/tcplfit2-vignette.Rmd

```diff
@@ -362,3 +362,109 @@ The output table resulting from the previous code chunk is the same format as th
     concRespPlot(res_wide[i,],ymin=-50,ymax=50)
   }
 ```
+
+## Example 5: How can you censor BMD
+When doing curve fitting, BMD is returned for curve with hit-call > 0. Users have the option to "censor" the BMD when it occurs above or below an experimental dose or other specified values. This section provides examples of how to do it in situations of when the BMD is lower or higher than the minimum or maximum concentration in the data with arguments `bmd_low_bnd` and `bmd_up_bnd`. 
+
+First, let's look at situation one: when BMD is below the lowest concentration tested. The following code chunk loads data of a sample from `mc3` and removes data in some of the low dose groups so it simulates a curve that takes a huge lift in the first couple dose groups. In this situation, the BMD is more likely to be a lower dose that are not in the data. 
+
+```{r example 5 lower}
+# load example data
+spid <- unique(mc3$spid)[94]
+temp <- mc3[is.element(mc3$spid,spid),]
+
+# The data file has stored concentration in log10 form, so fix that
+conc <- 10**temp$logc
```

Comment:
Make sure to put a comment at the end of the conc line since this is likely not a common command for 10^temp$logc.  You can just put something line "# back-transforming concentrations on log10 scale".